modules/spring-config/src/main/java/org/mule/runtime/config/internal/DefaultExpressionManagerFactoryBean.java

```diff
@@ -34,6 +34,8 @@ public ExtendedExpressionManager getObject() throws Exception {
     DefaultExpressionManager delegate = new DefaultExpressionManager();
     muleContext.getInjector().inject(delegate);
 
+    delegate.initialise();
```

Comment:
the initailization should happen at a stage after the creation of the object by the factory